hack/run-openshift.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OS_ROOT}/hack/common.sh"
+
+platform=$(os::build::host_platform)
+
+${OS_OUTPUT_BINPATH}/${platform}/openshift $@
```

Comment:
Rather than have a separate script doing this, can you compute the path directly in the makefile?  I don't really like having a script that runs one command.